core/cache/application.go

```diff
@@ -23,10 +22,11 @@ func newApplication(metrics *ControllerGauges, hub *pubsub.SimpleHub) *Applicati
 
 // Application represents an application in a model.
 type Application struct {
+	*Entity
```

Comment:
Any particular reason to use a pointer? Normally in Go we try not to unless necessary. I don't see why it would be necessary to have the pointer here.

Reply:
Originally I wanted to use `*Entity` for the `Check()` method argument, but I instead went for the cleaner `stabler` (horrid name I know!). I'll fix it now.